lua/plenary/path.lua

```diff
@@ -474,20 +474,77 @@ function Path:rename(opts)
   return status
 end
 
+--- Copy files or folders.
+---@param opts table: options to pass to toggling registered actions
+---@field destination string|Path: target file path to copy to
+---@field recursive bool: whether to copy folders recursively (default: true)
+---@field override bool: whether to override files (default: true)
+---@field interactive bool: confirm if copy would override; precedes `override` (default: false)
+---@field respect_gitignore bool: skip folders ignored by all detected `gitignore`s (default: true)
+---@field hidden bool: whether to add hidden files in recursively copying folders (default: true)
+---@return table {[Path of destination]: bool} indicating success of copy; nested tables constitute sub dirs
 function Path:copy(opts)
   opts = opts or {}
+  opts.recursive = F.if_nil(opts.recursive, false, opts.recursive)
```

Comment:
docs say recursive is true on default. implementation says false.

Reply:
I've opted to `recursive=false` by default such that it's more in line with `cp`.

Reply:
Yeah following `cp` is a good idea. I just "pointed" out minor inconsistencies in documentation and implementation.

Thanks for cleaning it up